packages/env/lib/wordpress.js

```diff
@@ -51,11 +52,21 @@ async function checkDatabaseConnection( { dockerComposeConfigPath, debug } ) {
  * @param {Object}        spinner     A CLI spinner which indicates progress.
  */
 async function configureWordPress( environment, config, spinner ) {
+	let wpVersion = '';
+	try {
+		wpVersion = readWordPressVersion( config.env[ environment ].coreSource, spinner, config.debug );
+	} catch ( err ) {
+		// Ignore error.
+	}
+
 	const installCommand = `wp core install --url="${ config.env[ environment ].config.WP_SITEURL }" --title="${ config.name }" --admin_user=admin --admin_password=password --admin_email=wordpress@example.com --skip-email`;
 
 	// -eo pipefail exits the command as soon as anything fails in bash.
 	const setupCommands = [ 'set -eo pipefail', installCommand ];
 
+	// WordPress versions below 5.1 didn't use proper spacing in wp-config.
+	const configAnchor = wpVersion && semver.lt( wpVersion, '5.1' ) ? `"define('WP_DEBUG',"` : `"define( 'WP_DEBUG',"`;
```

Comment:
Does this actually work?

WordPress major versions like `5.1` do not follow semver, so `semver.lt( wpVersion, '5.1' )` actually throws an error when I try it on RunKit.

In addition to that, adding the `semver` dependency increases the bundle size by 31
kB [according to Bundlephobia](https://bundlephobia.com/package/semver@7.5.4). That's a lot for just a single "below 5.1" check.

Given these two issues I suggest adding a simpler check that doesn't use semver and supports WP version numbers.

Something simple like this could work:

```
const versionParts = wpVersion ? wpVersion.split( '.' ) :[];
const configAnchor = versionParts && versionParts[0] <= 5 && versionParts[1] <= 1 ? `"define('WP_DEBUG',"` : `"define( 'WP_DEBUG',"`;

----

Aside:

The PR title says WP 5.4, but the check here is for 5.1 — which one is it?

Reply:
Sounds good, we can remove `semver` in favor of a simple function to do this ourselves.

> The PR title says WP 5.4, but the check here is for 5.1 — which one is it?

See the PR description: it's both depending on the concrete problem, so overall it's 5.4, but part of the checks are only relevant before 5.1.

Reply:
I've updated this to use a simple custom function in https://github.com/WordPress/gutenberg/pull/55864/commits/210301815193995d9f117392ebdf9a55db101b77.